kafka.go

```diff
@@ -127,12 +127,12 @@ func setLogLevel(logMode string) {
 }
 
 // newKafkaClient create a new instance of KafkaClient
-func newKafkaClient(brokers []string, prefix string, config ...*BrokerConfig) (*KafkaClient, error) {
+func newKafkaClient(brokers []string, balancer kafka.Balancer, prefix string, config ...*BrokerConfig) (*KafkaClient, error) {
 	logrus.Info("create new kafka client")
 
 	writerConfig := &kafka.WriterConfig{
 		Brokers:  brokers,
-		Balancer: &kafka.LeastBytes{},
+		Balancer: balancer,
```

Comment:
it is handled by the segmentio already, if the balancer isn't set then by default it will use roundrobin. but to double ensure, its also set in this lib. feedback addressed.